### DISCUSSION TITLE
When using in multiple package it's better as a peer dependency

### DISCUSSION THREAD

**Discussion by huvber** [vercel/swr#900] [Q&A]
Hello, 
first thanks a lot for making this library. It's amazing. 
I have a doubt. We have a big project divided in small inter-dependent modules. My question is:
In this scenario it's better to put swr as a peer dependency or it's totally fine to have it as dependency of the different packages?

Thanks in advance

**Comment by koba04** [comment]
It seems to depend on your project structure. But in general, if you use the same version of `swr`, bundlers only include single version of `swr` in a bundle file even you put `swr` in each dependency. I recommend trying it and analyze the result.
